config/dynamicconfig/development_es.yaml

```diff
@@ -2,6 +2,8 @@ frontend.enableClientVersionCheck:
   - value: true
 system.advancedVisibilityWritingMode:
   - value: "on"
+system.advancedVisibilityMigrationWritingMode:
```

Comment:
are we going to change this property to make it more descriptive/extensible in follow up PR?

Reply:
Yes, it is addressed in https://github.com/cadence-workflow/cadence/pull/6560, I haven't tested the changes there, will add to the queue later